pkgs/by-name/sw/swhkd/package.nix

```diff
@@ -0,0 +1,28 @@
+{ lib
+, pkgs
+, stdenv
+, fetchFromGitHub
+, rustPlatform
+}:
+let
+  name = "swhkd";
+  version = "30f25b5bf99df5f16d91b51a7bd397c1de075085";
```

Comment:
Convention for version is `1.2.1+unstable=YYYY-MM-DD`, with date corresponding to the commit date. Also consider asking upstream for proper tag.